### ISSUE TITLE
[BUG] Foundry Restart/UI Change on Fudge Creation

### ISSUE THREAD

**Issue by chordialChasma** [UranusBytes/foundry-die-hard#38]
I'll try to keep this brief.
I've tested this in a new world with nothing installed in Foundry Version 10 Build 268.

![image](https://user-images.githubusercontent.com/44590508/213926907-8ca98bbc-b803-45fd-a62b-b7e95a3e8b69.png)

When attempting to create a new Fudge, Foundry restarts for me and extends the URL with additional arguments that appear to be for the fudge itself. This also causes the UI to get weird (seen on the right menu).

![image](https://user-images.githubusercontent.com/44590508/213926820-2f8fded4-6621-40ba-a2e1-2dd28ee47a4f.png)

libwrapper is up to date, and in this new world, all other modules have been disabled. I was using this a few weeks ago without any issue, and the other Fudges that already exist in the world I've been using are still able to be turned on/off.

I haven't updated the foundry version past 10.268 since it released.

TLDR:
Fudges aren't able to be made, cause foundry to restart/ui to get weird. Tested this on a new world with the same results, and the module was working fine a few weeks ago.

**Comment by UranusBytes** [comment]
For some reason, this code isn't working any more

https://github.com/UranusBytes/foundry-die-hard/blob/fad142fd237e3f755b8f6f1900c4d65a19650ba6/scripts/classes/DieHardFudgeDialog.js#L416-L419

A temp hack is to edit this file and comment out the if statement so it always runs the super._onSubmit.  It's going to take further testing and investigation to know implications...

```
    // Hack for v10
    //if (event.path?.length > 0) {
      super._onSubmit(event, __namedParameters)
    //}
```

**Comment by Chronolegionnaire** [comment]
Just a heads up, it doesn't look like commenting out that bit of code works around the issue anymore either. If it helps at all, when you press "create fudge" foundry redirects to url/game?fudgeFormula=&fudgeTimes=&create= 

obciously with url replaced with whatever your foundry instance is hosted from.

Changing the different settings results in the redirect changing for example choosing players or changing the dice value will show up in the url string as well.

**Comment by thortain** [comment]
I'm also experiencing this issue

**Comment by UranusBytes** [comment]
@thortain What version are you running?  The latest at 0.1.2?  Are there any errors in your browser console?

**Comment by thortain** [comment]
the code edit listed above fixed the issue, im not sure if theres an update i havent tried yet. the far more pressing bug is the fact it breaks a lot of rolls when using MidiQol
